save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -203,14 +206,24 @@ class DockerService(private val configProperties: ConfigProperties) {
 
     @Suppress("UnsafeCallOnNullableType")
     private fun copyTestSuitesToResourcesPath(testSuitesForDocker: List<TestSuiteDto>, destination: File) {
+        log.info("Copying suites ${testSuitesForDocker.map { it.name }} into $destination")
         testSuitesForDocker.forEach {
             val standardTestSuiteAbsolutePath = File(configProperties.testResources.basePath)
                 // tmp directories names for standard test suites constructs just by hashCode of listOf(repoUrl); reuse this logic
                 .resolve(File("${listOf(it.testSuiteRepoUrl!!).hashCode()}")
                     .resolve(File(it.propertiesRelativePath).parent)
                 )
+            log.debug("Copying suite ${it.name} from $standardTestSuiteAbsolutePath into $destination/...")
             standardTestSuiteAbsolutePath.copyRecursively(destination.resolve("${it.name}_${it.propertiesRelativePath.hashCode()}_${Random.nextInt()}"))
         }
+        // orchestrator is executed as root (to access docker socket), but files are in a shared volume
+        val lookupService = destination.toPath().fileSystem.userPrincipalLookupService
+        destination.walk().forEach {
+            Files.getFileAttributeView(it.toPath(), PosixFileAttributeView::class.java, LinkOption.NOFOLLOW_LINKS).apply {
+                setGroup(lookupService.lookupPrincipalByGroupName("cnb"))
+                owner = lookupService.lookupPrincipalByName("cnb")
```

Comment:
May be use one style here?

```suggestion
                setOwner(lookupService.lookupPrincipalByName("cnb"))
```

Strange, that we can't use `group = ....` here, like we does for `owner`

Reply:
Yeah, that's because `owner` has both setter and getter, while `group` has only a setter. So Kotlin can't implement syntactic sugar with property access for group.

Reply:
I thought it would be a compiler warning, but it's only IDEA's warning, so I'll change both to explicit setters